web/src/main/kotlin/no/nav/su/se/bakover/web/routes/kontrollsamtale/KontrollsamtaleRoutes.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.su.se.bakover.web.routes.kontrollsamtale
+
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import no.nav.su.se.bakover.domain.Brukerrolle
+import no.nav.su.se.bakover.domain.NavIdentBruker
+import no.nav.su.se.bakover.service.kontrollsamtale.KontrollsamtaleService
+import no.nav.su.se.bakover.service.kontrollsamtale.KunneIkkeKalleInnTilKontrollsamtale
+import no.nav.su.se.bakover.web.Resultat
+import no.nav.su.se.bakover.web.errorJson
+import no.nav.su.se.bakover.web.features.authorize
+import no.nav.su.se.bakover.web.features.suUserContext
+import no.nav.su.se.bakover.web.routes.Feilresponser
+import no.nav.su.se.bakover.web.svar
+import no.nav.su.se.bakover.web.withBody
+import java.util.UUID
+
+internal fun Route.kontrollsamtaleRoutes(
+    kontrollsamtaleService: KontrollsamtaleService
+) {
+    authorize(Brukerrolle.Saksbehandler) {
+        post("kontrollsamtale/kallInn") {
+            data class Body(
+                val sakId: UUID,
```

Comment:
Hadde kanskje sett for meg at man skulle sendt med en frist/periode/dato av et eller annet slag for å si noe om "når" kontrollsamtalen skal forekomme?

Reply:
Mulig dette kommer på et tidspunkt.